portality/settings.py

```diff
@@ -683,8 +683,16 @@
         "suggestion" : {
             "auth" : True,
             "role" : "admin",
+            "query_filters" : ["not_update_request"],
```

Comment:
I think this is fine for the route, but the actual method behind `not_update_request` is still differentiating between update requests and new applications in the old way, it needs to look at `admin.application_type` for either new applications or update requests (see `constants.APPLICATION_TYPE_NEW_APPLICATION` and the related UR one).